Hestia/Hestia/frontend/DevicesScreen/EditDevice/UIViewControllerEditDeviceName.cs

```diff
@@ -46,12 +46,62 @@ public override void ViewDidLoad()
             changeNameField.Frame = new CGRect(110, 10, this.View.Bounds.Width - 125, 31);
             changeNameField.Placeholder = device.Name;
             rectangle.AddSubview(changeNameField);
-            View.BackgroundColor = Globals.DefaultLightGray;
+
+
```

Comment:
Maybe we should rework this ViewController to make it more flexible and handle other kind of required info items, because some plugins have required info fields..